src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -256,7 +256,9 @@
             aggregate-fields)])))})
 
 (def ^:private PERSISTENCE-KEY "library-page")
-(def ^:private VERSION 3)
+(def ^:private VERSION 2)
+(def ^:private TABLE_VERSION 1)
+
```

Comment:
i'll fix this and remove table version. this was some weird variant from when i was adding a version number for the table then matt refactored the table in a better way. i added a comment to 680 to add some constants for the table key and version if you want.